package.json

```
@@ -16,11 +16,17 @@
     "unicop <its.op.the.unicorn@gmail.com>",
     "neriyarden"
   ],
-  "keywords": ["eslint", "typescript", "@typescript-eslint/eslint-plugin", "migrate", "@typescript-eslint/parser"],
+  "keywords": [
+    "eslint",
+    "typescript",
+    "@typescript-eslint/eslint-plugin",
+    "migrate",
+    "@typescript-eslint/parser"
+  ],
   "scripts": {
     "build": "tsc -p tsconfig.json",
     "test": "jest",
-    "prebuild": "rimraf build && lint",
+    "prebuild": "rimraf build && npm run lint",
```

to avoid duplciate run of lint, during the `build` job because of the `npm hooks`, lets create a new npm script
call: `build:only`, which is disconnected from the `prebuild` hook.
then:

```suggestion
    "build:only": "rimraf build && tsc -p tsconfig.json",
    "build": "npm run build:noly",
    "test": "jest",
    "prebuild": "npm run lint",
    ....
```
then on the `build` job in the `.circleci.yml` make sure it is running `build:only` instead of `build`.